lib/rack/request_police/middleware.rb

```diff
@@ -25,6 +25,25 @@ def call(env)
             if %w(POST PATCH DELETE).include?(env['REQUEST_METHOD'])
               request_params.merge!('data' => utf8_input(env))
             end
+
+            headers = Rack::RequestPolice.headers.inject({}) do |result, header_hash|
+              header_name = header_hash.fetch(:original_header_name)
+              transformation = header_hash.fetch(:transformation)
+
+              fallback_value = header_hash.fetch(:fallback_value)
+              storage_name = header_hash.fetch(:storage_name)
+
+              header_value = env[header_name]
+              # apply transformations to header_value, fail silently on any exception
+              header_value = transformation.call(header_value) rescue header_value
```

Comment:
We're transforming a string there, so personally I would rather get 500 instantly than to swallow exception silently (bad experience with too much rescue :trollface: ) - but within current implementation this is indeed not possible - `transformation` is always called, even if header is missing (you transform `nil`, rescue it if something bad happens (that returns `nil` again) and then fallback to _fallback value_ (I hope I didn't mess up anything, but it seems about right). It might be a personal taste, but I will allow myself to modify this a little :wink: 

Except that - thanks again for PR :+1: I hope I will manage to release new version this week, this is probably a good time to add a changelog too ;-).

Cheers :heart: